constraint/pkg/client/client.go

```diff
@@ -682,6 +712,10 @@ func (c *Client) Review(ctx context.Context, obj interface{}, opts ...drivers.Qu
 			continue
 		}
 
+		for i := range resp.Results {
+			resp.Results[i].EnforcementAction = enforcementActionsByTarget[target][resp.Results[i].Constraint.GetName()]
```

Comment:
ensure enforcementActionsByTarget is not nil, and enforcementActionsByTarget[target] is not nil

Reply:
add tests for these cases

Reply:
> ensure enforcementActionsByTarget is not nil, and enforcementActionsByTarget[target] is not nil

This will never be the case as `enforcementActionsByTarget` is initialized by `make()` and without any `target` this line will not get executed at all. I added a check to verify the entry in the map to make sure just in case